intl/msg_hash_us.h

```diff
@@ -1809,7 +1809,7 @@ MSG_HASH(
    )
 MSG_HASH(
    MENU_ENUM_SUBLABEL_MENU_DRIVER,
-   "Menu driver to use. Requires restart."
+   "Menu driver to use. (Restart required)"
```

Comment:
s/Menu driver to use. (Restart required)/Menu driver to use (Restart required)./